<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowShowScene</class>
 <widget class="QMainWindow" name="MainWindowShowScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> VirtualRobot - ShowScene</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Scene/Object File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ExportPDF">
       <property name="toolTip">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>191</width>
        <height>31</height>
       </rect>
        <string>Export to PDF format</string>
       </property>
       <property name="text">
        <string>Export PDF</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="cam2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>400</width>
         <height>260</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>11</width>
         <height>0</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
    </widget>
    </item>
   </layout>
</widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
